src/python/turicreate/test/test_style_transfer.py

```diff
@@ -292,6 +292,17 @@ def test_save_and_load(self):
             self.test_get_styles_success()
             print("Get styles passed")
 
+    def test_state(self):
+        model = self.model
+        fields = model.__proxy__.list_fields()
+        assert "model" in fields
```

Comment:
In unit tests `self.assertTrue` is usually preferable to just `assert`. I believe using `assert` will cause failures to show up as errors which isn't ideal.

Reply:
Done, thanks!